src/ray/object_manager/object_manager.cc

```diff
@@ -385,6 +397,12 @@ ray::Status ObjectManager::Wait(const std::vector<ObjectID> &object_ids,
     }
   }
 
+  return ray::Status::OK();
```

Comment:
If this method always returns `OK`, I would make it `void`.

Reply:
It actually also returns `ray::Status::NotImplemented` currently if `wait_local=true`.